<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>1031</width>
     <height>661</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>点餐</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>虹梦点餐</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>换肤</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="jiansuo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>请输入菜名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_jiansuo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>550</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>检索</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>精品热菜</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>390</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>精品冷菜</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>711</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>当前选择的是:</string>
     </property>
    </widget>
    <widget class="QLabel" name="xuancai">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>651</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>酒水饮料</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>480</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>海洋盛宴</string>
     </property>
    </widget>
    <widget class="QLabel" name="caiyulan">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="changeduser">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>600</y>
       <width>89</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>切换账户</string>
     </property>
    </widget>
    <widget class="QLabel" name="ccPri">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>20</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>20</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>当前价格:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>下单</string>
    </attribute>
    <widget class="QLabel" name="xiadan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>601</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>加载中.....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>请填写客人信息</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cusname">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>150</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="custel">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>190</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="zhuohao">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="taste">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>270</y>
       <width>113</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>客户姓名</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>150</y>
       <width>117</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>先生</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>150</y>
       <width>117</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>小姐</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>190</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>联系方式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>441</x>
       <y>230</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>桌号（外卖0）</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>310</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>口味备注</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>430</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>下单时间</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>350</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="xuanhao">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>查看当前已选</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>341</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>结算</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>580</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>打印小票</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cacl">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>580</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>结账</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>520</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="dayin">
     <property name="geometry">
      <rect>
       <x>461</x>
       <y>520</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="kehuxinxi">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>261</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="xianshijiesuan">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>431</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QLabel" name="lianjie">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>正在检测...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库状态:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>结算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpdia">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>10</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
